### DISCUSSION TITLE
source code violation

### DISCUSSION THREAD

**Discussion by FormlessFang** [GiorgioBrux/nitro-sniper-enhanced#134] [Q&A]
tried to deploy on heroku and it violates their acceptable use now, anyway to bypass this?

**Comment by GiorgioBrux** [comment]
Apologies for not answering earlier, github doesn't send me notifications for discussions.
Did you try the steps listed [here](https://github.com/GiorgioBrux/nitro-sniper-enhanced#heroku-recommended)?

**Reply by FormlessFang** [reply]
Yes I have but it still says it violates their acceptable use.

**Reply by GiorgioBrux** [reply]
Sorry for the late answer. You can try making a repo from scratch and then committing the sniper files. That should work.

**Reply by FormlessFang** [reply]
How would I go about doing that?

**Reply by GiorgioBrux** [reply]
You should be able to use the github desktop client without touching the cli.

**Reply by FormlessFang** [reply]
Sorry about the late response, didn't receive an email or notification that you had replied. Could you give me a step by step instructions on how to do this if you can?

**Reply by GiorgioBrux** [reply]
Hi. I'm very very very sorry, didn't receive an email/notification either. Did you manage to do it eventually?

**Reply by FormlessFang** [reply]
Hi, It's okay. I couldn't figure it out so I gave up.
